msteams-platform/graph-api/meeting-transcripts/fetch-id.md

```diff
@@ -532,3 +532,4 @@ You can try the following code sample for a bot app:
 
 - [Apps for Teams meetings and calls](../../apps-in-teams-meetings/teams-apps-in-meetings.md)
 - [API reference for the Bot Framework Connector service](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference)
+- [Get change notifications for Teams meeting call updates](/graph/changenotifications-for-onlinemeeting)
```

Comment:
Unclear what is change notifications. Change hyperlink text based on "_subscribe to call started/ended and call roster updates for Microsoft Teams online meetings._" from the linked article.